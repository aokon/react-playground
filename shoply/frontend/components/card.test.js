import React from 'react'
import renderer from 'react-test-renderer'
import Card from './card'

test('Footer Conponent', () => {
  const category = {
    id: '123123-123123-23234',
    attributes: {
      name: 'Category 1'
    }
  }
  const component = renderer.create(
    <Card model={category}
       url={`/category?id=${category.id}`}
       url_map={`/category/${category.id}`} />
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})