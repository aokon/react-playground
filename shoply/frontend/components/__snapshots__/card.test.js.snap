// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Conponent 1`] = `
<div
  className="col s12 m6 category-card"
>
  <div
    className="card blue-grey darken-1"
  >
    <div
      className="card-content white-text"
    >
      <span
        className="card-title"
      >
        Category 1
      </span>
      <p
        className="card-content"
      >
        I am a very simple card. I am good at containing small bits of information. I am convenient because I require little markup to use effectively.
      </p>
    </div>
    <div
      className="card-action"
    >
      <a
        href="/category?id=123123-123123-23234"
        onClick={[Function]}
      >
        Show more
      </a>
    </div>
  </div>
</div>
`;
